viewing_party/party.py

```diff
@@ -0,0 +1,181 @@
+#  WAVE 1 #
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    else: 
+        return None
```

Comment:
This works great but you could just return the title. genre, and rating without doing movie_dict = {}
```suggestion
def create_movie(movie_title, genre, rating):
    if movie_title == None or genre == None or rating == None:
        return None
    else:
        return {
        "title":title, 
        "genre":genre, 
        "rating":rating
        }```